WalletWasabi.Fluent/Views/CoinControl/Core/Cells/LabelsCellView.axaml

```diff
@@ -0,0 +1,39 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:controls="clr-namespace:WalletWasabi.Fluent.Controls"
+             xmlns:cells="clr-namespace:WalletWasabi.Fluent.ViewModels.CoinControl.Core.Cells"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:Class="WalletWasabi.Fluent.Views.CoinControl.Core.Cells.LabelsCellView"
+             x:DataType="cells:LabelsCellViewModel"
+             x:CompileBindings="True">
+
+  <UserControl.Resources>
+    <SolidColorBrush x:Key="PrivacyLevelNoneBrush" Color="{DynamicResource TextForegroundColor}" />
+  </UserControl.Resources>
+
+  <UserControl.Styles>
+
+    <Style Selector="controls|LabelsItemsPresenter.NonPrivate">
+      <Setter Property="BorderBrush" Value="{DynamicResource PrivacyLevelNoneBrush}" />
+      <Setter Property="Foreground" Value="{DynamicResource PrivacyLevelNoneBrush}" />
+    </Style>
+    <Style Selector="controls|LabelsItemsPresenter.SemiPrivate">
+      <Setter Property="BorderBrush" Value="{DynamicResource PrivacyLevelMediumBrush}" />
+      <Setter Property="Foreground" Value="{DynamicResource PrivacyLevelMediumBrush}" />
+      <Setter Property="MaxLabelWidth" Value="1000" />
```

Comment:
Why we want such long labels here, wouldn't it cause layout issues in TDG ?

Reply:
Why 1000 ?

Reply:
Let me explain the problem:

The MaxLabelWidth is set in the default style:

![image](https://user-images.githubusercontent.com/3109851/203041375-f02d340f-ed95-4600-ac89-51b8fcbf9494.png)

But when it comes to well-known labels (not real labels) like "Private Coins" "Unknown People" and so, we don't want the character ellipsis to be applied. Thus, I needed to override the value with a number high enough. This is the simplets solution, I've found, since `double.PositiveInfinity` can't be specified in XAML, AFAIK.

If there's a better solution, please tell me :)